src/foam/dao/AuthenticatedDAO.java

```diff
@@ -123,11 +122,12 @@ public void pipe_(foam.core.X x, foam.dao.Sink sink, foam.mlang.predicate.Predic
   }
 
   public Predicate augmentPredicate(X x, Predicate existingPredicate, String operation) {
-      return existingPredicate != null ?
-        AND(
-          HAS_PERMISSION(x, createPermission(operation)),
-          existingPredicate
-        ) :
-        HAS_PERMISSION(x, createPermission(operation));
+    boolean remove = operation.equals("delete");
+    return existingPredicate != null ?
```

Comment:
Doesn't this need to change to use the new MLang's?

Reply:
this will get removed with the authenticatedDAO once everything is migratde right?

Reply:
ok